src/components/modals/FlipInputModal.tsx

```diff
@@ -251,9 +249,11 @@ export class FlipInputModalComponent extends React.PureComponent<Props, State> {
   }
 }
 
+const deviceHeight = Dimensions.get('window').height
+
 const getStyles = cacheStyles((theme: Theme) => ({
   hackContainer: {
-    flex: deviceHeight <= MINIMUM_DEVICE_HEIGHT ? 1 : 0
+    flex: deviceHeight <= 580 ? 1 : 0
```

Comment:
Why was the constant removed? I like that it described the value. Can we at least move it to the top of this file?

Reply:
The constant did not describe anything. If I gave you a constant named "MINIMUM_DEVICE_HEIGHT" and no context, what would you guess it meant? Probably "the app won't run on phones less than 580px tall", or something incorrect like that.

The CSS section is always full of numbers. That's what they are for - to describe how things look. Given it's new home in the CSS section, the number is no longer misleading. It's perfectly obvious that the modal flexes only if the phone is below 580px in size now.

Reply:
Fair enough.